lib/routes/utils/oauth-mint.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const jwtool = require('fxa-jwtool')
+const Pool = require('../../pool')
+
+// From the Firefox source code:
+// Having this parameter be short has limited security value and can cause
+// spurious authentication values if the client's clock is skewed and
+// we fail to adjust. See Bug 983256.
+const ASSERTION_LIFETIME = 1000 * 3600 * 24 * 365 * 25 // 25 years in ms.
+const CERT_LIFETIME = 1000 * 3600 * 5 // 5 hours in ms.
+
+const CLIENT_ID = '5882386c6d801776' // Desktop CLIENT_ID, is this ok?
```

Comment:
Let's allocate a different oauth client id for this.  Or perhaps we can re-use the one that's already used by content-server - @vladikoff thoughts?

Reply:
I'm saying let's generate a new one